pkg/libcoap/Makefile

```diff
@@ -0,0 +1,15 @@
+PKG_NAME=libcoap
+PKG_URL=https://github.com/mrdeep1/libcoap
```

Comment:
Ideally you wouldn't fork the project but move patches that are needed for RIOT to  
`pkg/libcoap/patches` folder where they are applied automatically on checkout.
You should also try to upstream your changes where possible.

That way updating to future versions of the library will become easier and we are not stuck with an old fork.

Reply:
Agreed.  I was using my fork as I found that to be the easiest to test out the changes I found I needed to handle 16 bit integers along with other things that the different compilers were throwing up to get Murdock to complete.  My short term plan was to get the libcoap specific changes merged into libcoap project and update PKG_URL / PKG_VERSION as appropriate,

That said, I will update this so that it is a set of patches shortly so that we can move quickly forward here and update the libcoap project in slow time (which will in he next few days).

Reply:
Updated as a patch.